ligo/types.mligo

```diff
@@ -38,50 +92,95 @@ type position_map = (position_index, position_state) big_map
 type metadata_map = (string, bytes) big_map
 
 type storage = {
-    liquidity : nat ; (* virtual liquidity, the value L for which the curve locally looks like x * y = L^2 *)
-    sqrt_price : nat ; (* square root of the virtual price, the value P for which P = x / y *)
-    i_c : int ;(* the highest tick corresponding to a price less than or equal to sqrt_price^2, does not necessarily corresponds to a boundary *)
-    lo : tick_index ; (* the highest initialized tick lower than or equal to i_c *)
+    (* Virtual liquidity, the value L for which the curve locally looks like x * y = L^2. *)
+    liquidity : nat ;
+    (* Square root of the virtual price, the value P for which P = x / y. *)
+    sqrt_price : nat ;
+    (* Index of the highest tick corresponding to a price less than or equal to sqrt_price^2,
+        does not necessarily corresponds to a boundary.
+    *)
+    i_c : int ;
+    (* The highest initialized tick lower than or equal to i_c. *)
+    lo : tick_index ;
+    (* The total amount of fees that have been earned per unit of virtual liquidity (L),
+        over the entire history of the contract.
+    *)
     fee_growth : balance_nat ;
```

Comment:
This is also a controversial point, along with the points mentioned at the top of PR. Looks like adding `_global` suffix is an unnecessary boilerplate because any separate field in `storage` is "global".

There are also other fields (like `i_c`/`tick`), for which we probably don't want to follow the convention taken in the article because it is inconvenient in code. Probably we should discuss such corner cases and see which convention do we want to follow eventually, and then I'll apply it to all the remaining fields.

What do you think?

> Also, looks like this point fromTCFMM-8 hasn't been addressed yet:

I address this in #7. In the presence of newtype wrapper, TCFMM-8 does not require to rename the field.

Reply:
> What do you think?

Sounds good, agreed, let's keep it as `fee_growth` then!

> I address this in #7. In the presence of newtype wrapper, TCFMM-8 does not require to rename the field.

Ok I'll have a look at #7 then! :+1: